x/axelarnet/keeper/msg_server.go

```diff
@@ -470,7 +470,19 @@ func (s msgServer) RetryIBCTransfer(c context.Context, req *types.RetryIBCTransf
 		return nil, fmt.Errorf("chain %s is not activated", chain.Name)
 	}
 
-	err := s.ibcK.SendIBCTransfer(ctx, t)
+	lockableAsset, err := s.nexus.NewLockableAsset(ctx, s.ibcK, s.bank, t.Token)
+	if err != nil {
+		return nil, err
+	}
+
+	err = lockableAsset.UnlockTo(ctx, types.AxelarIBCAccount)
+	if err != nil {
+		return nil, err
+	}
+	// Notice: all IBC transfer are routed from AxelarIBCAccount after v1.1
+	t.Sender = types.AxelarIBCAccount
```

Comment:
hmm, not the ideal solution, should we consider migrating the transfers, or at least add a TODO/task?

Reply:
I will migrate